ports/opencv4/vcpkg.json

```
@@ -30,6 +30,12 @@
         "ade"
       ]
     },
+    "aravis": {
+      "description": "aravis",
+      "dependencies": [
+        "aravis"
```

Enable opt-out from usb.
```suggestion
        {
          "name": "aravis",
          "default-features": false
        }
```

---

then it might be better to leave `usb` away as a default feature from aravis? i now went with that option to stay aligned with aravis (though i think they make it dependent on the availability of the library by default since they set it to `auto`)

---

this is not meant for aravis features, but for opencv features. It just means that if you ask for aravis you don't depend on all the default features

---

vcpkg will install aravis' default feature *by default*. This change *enables* the user to *explicitly* request installation of aravis without default features.